dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/UnresolvableArtifactProblem.java

```diff
@@ -41,6 +42,18 @@ public UnresolvableArtifactProblem(Artifact artifact) {
   public UnresolvableArtifactProblem(List<DependencyNode> dependencyPath) {
     super(dependencyPath.get(dependencyPath.size() - 1).getArtifact(), dependencyPath);
   }
+  
+  /**
+   * Problem when Maven cannot resolve the artifact at the leaf of {@code dependencyPath} in a
+   * dependency tree.
+   * 
+   * @param artifact dependency that can't be found
+   * @param path to the unfound artifact
```

Comment:
```suggestion
   * @param path path to the unavailable artifact
```

Reply:
Because the artifact is held by the last element of path, you don't need the constructor.